sympy/calculus/util.py

```diff
@@ -1290,21 +1308,20 @@ def __rtruediv__(self, other):
 
     @_sympifyit('other', NotImplemented)
     def __pow__(self, other):
-        from sympy.functions.elementary.miscellaneous import real_root
         if isinstance(other, Expr):
             if other is S.Infinity:
                 if self.min.is_extended_nonnegative:
                     if self.max < 1:
                         return S.Zero
-                    if self.min > 1:
+                    if self.min >= 1:
```

Comment:
`min` should be strictly greater than 1. If 1 is an accumulation value of a function `f(t)` at `a`, then there may be a sequence `t_n` tending to `a` such that `f(t_n) = 1` for all `n`. Then also `f(t_n)**n = 1`, so 1 is also an accumulation value for exponent `oo`.

Reply:
Should B(1,2)**oo be B(0, oo) or B(1, oo)?

Reply:
`B(0, oo` as the function `f` having accumulation points in `[1, 2]` may take values less than 1. In the example above I had chosen `f(t_n) = 1` for simplicity, but we could also have `f(t_n) = 1 - 1/n`. If `g(t_n)` tends to `oo` so that `g(t_n)/n` also tends to `oo` (e.g,, `g(t_n) >= n**2`), then `f(t_n)**g(t_n) = ((1 - 1/n)**n)**(g(t_n)/n) -> (1/E)**oo = 0`.

Reply:
Maybe starting to get the idea. Is `sin(oo)` = [-1,1] because the curvature at pi/2 is concave up or down so whether you approach from the right or left, the limit is 1?

Reply:
`sin(x)` might be a slightly misleading example as its values (on the real axis) are in the interval `[-1, 1]`. A better example would be `f(x) = (1 + 1/x)*sin(x)` that takes repeatedly values not in the interval `[-1, 1]` though its accumulation values at `oo` are there. 

For each `b` in `[-1, 1]`, there is a real `c` such that  `sin(c) = b`. Then `x_n = c + 2*pi*n` is a sequence tending to `oo` such that `f(x_n) -> b`, so `b` is an accumulation value of `f` at `oo`.